src/components/File/NewNote.tsx

```diff
@@ -15,83 +12,45 @@ interface NewNoteComponentProps {
 
 const NewNoteComponent: React.FC<NewNoteComponentProps> = ({ isOpen, onClose }) => {
   const { openContent: openTabContent } = useWindowContentContext()
-
-  const [fileName, setFileName] = useState<string>('')
-  const [errorMessage, setErrorMessage] = useState<string | null>(null)
-
   const { currentlyOpenFilePath } = useFileContext()
 
-  useEffect(() => {
-    if (!isOpen) {
-      setFileName('')
-      setErrorMessage(null)
-    }
-  }, [isOpen])
+  const createNewFile = async () => {
```

Comment:
we can get of the newnote component fully right? this function can just go in the iconssidebar where the icon for new note is pressed and we can fully delete the newnotecomponent